example/ios/Podfile.lock

```diff
@@ -432,7 +432,7 @@ PODS:
     - Yoga (~> 1.14)
 
 DEPENDENCIES:
-  - attentive-react-native-sdk (from `../..`)
+  - "attentive-react-native-sdk (from `../node_modules/@attentive-mobile/attentive-react-native-sdk`)"
```

Comment:
is importing this from the node_modules expected? why do we want to do that instead of importing directly?

Reply:
Unsure, but running `yarn install` kept forcing the change, so I assume it's a react native hook that's doing something and this is how it internally deals with links.